feat: add links to offchain::ipfs and http tutorial

niklaslong opened the pull request:
First quick prototyping, still missing a link for the http tutorial (will be added once merged). Adjusted the link colour to a slightly darker hue for contrast on the green. 

Closes #10.

![Screenshot 2020-10-30 at 16 43 46](https://user-images.githubusercontent.com/13221615/97726153-d8518500-1ac6-11eb-8723-ab22e03e558c.png)

niklaslong committed:
feat: add links to offchain::ipfs and http tutorial

aphelionz commented:
Awesome, can you include a screenshot in the PR description?

niklaslong commented:
I also (very quickly) played around with the colours on another branch, not sure if this is open for discussion (if it isn't feel free to ignore this 😉) but thought I'd share:

![Screenshot 2020-10-30 at 17 28 33](https://user-images.githubusercontent.com/13221615/97731896-a132a200-1acd-11eb-9020-4940f494af63.png)

aphelionz commented on index.html:
```diff
@@ -169,6 +169,14 @@ <h2>Getting Started</h2>
+    <section id="further-reading">
+      <h2>Further Reading</h2>
+      <p>These are some extra resources to get you started with specific use-cases of Rust IPFS:</p>
+      <p><a href="https://rs-ipfs.github.io/offchain-ipfs-manual/">The offchain::ipfs manual</a>: a project integrating IPFS and Substrate.</p>
```
Had you considered an `<ol>` here?

niklaslong replied:
I'd considered an `<ul>`, might be better to be honest 🤔

niklaslong replied:
![Screenshot 2020-10-30 at 18 54 33](https://user-images.githubusercontent.com/13221615/97740340-0a6be280-1ad9-11eb-9458-dc322a41d239.png)

aphelionz reviewed (approved):
LGTM, I think we can merge as-is. Just one very quick question.

niklaslong committed:
use ul instead of p for links

niklaslong commented:
I've added the future link to the getting started guide, we'll need to merge https://github.com/rs-ipfs/rust-ipfs/pull/420 as well. I've also changed to a `ul`.

aphelionz commented:
Oh! One more thing - either move that accessibility link "back to top" to under the further reading section, or add a new one there

niklaslong committed:
add missing accessibility link

niklaslong commented:
Ah, good catch—done ✅

aphelionz commented on index.html:
```diff
@@ -169,6 +169,19 @@ <h2>Getting Started</h2>
+      <div>
+        <ul>
+          <li><a href="https://rs-ipfs.github.io/offchain-ipfs-manual/">The offchain::ipfs manual</a>: a project integrating IPFS and Substrate.</li>
+          <!-- TODO: Add link once merged -->
```
Remove this comment and we'll me good to merge!

niklaslong replied:
Fixed 👍

niklaslong committed:
remove todo
